prusti-tests/tests/parse/pass/extern-spec/traits/foreign_mods.rs

```diff
@@ -0,0 +1,16 @@
+use prusti_contracts::*;
+
+#[extern_spec]
+extern "C" {
+    #[ensures(a >= b -> result == a)]
+    #[ensures(b >= a -> result == b)]
```

Comment:
```suggestion
    #[ensures(a >= b ==> result == a)]
    #[ensures(b >= a ==> result == b)]
```

Prefer this for Prusti specs.